src/algebra/module/localized_module.lean

```diff
@@ -999,6 +1004,80 @@ end
 
 end algebra
 
-end is_localized_module
+variables [module (localization S) M'] [is_scalar_tower R (localization S) M']
+/--
+- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
+- `(localization S) × M → N, (s, m) ↦ s • f m` is a tensor product.
+- In particular, there is an isomorphism between `localized_module S M` and
+- `(localization S) ⊗[R] M` given by `m/s ↦ (1/s) ⊗ₜ m`.
+--/
+theorem is_base_change :  is_base_change (localization S) f :=
+begin
+  apply is_base_change.of_lift_unique,
+  introsI,
+  have := is_localized_module.is_universal S f g
+  begin
+    intros s,
+    rw [module.End_is_unit_iff, function.bijective_iff_exists_unique],
+    intros q,
+    existsi (localization.mk 1 s • q),
```

Comment:
```suggestion
    use (localization.mk 1 s • q),
```
I think `use` is more commonly used than `existsi` -- is there a reason why you want to use `existsi`?